Cython/Utility/ModuleSetupCode.c

```diff
@@ -378,7 +378,9 @@
         #endif
     #endif
 #else
+#ifdef HAVE_STDINT_H
```

Comment:
How commonly is this macro defined?

Reply:
This macro is defined in python.h (at least from Python 2.5)

Reply:
OK, can you put this in a separate PR?